test/metadata-retrieval/tool.go

```diff
@@ -0,0 +1,30 @@
+package metadataretrieval
+
+import "github.com/src-d/regression-core"
+
+// NewToolMetadataRetrieval creates a Tool with metadata-retrieval parameters filled
+func NewToolMetadataRetrieval() regression.Tool {
+	return regression.Tool{
+		Name:        "metadata-retrieval",
+		GitURL:      "https://github.com/src-d/metadata-retrieval",
+		ProjectPath: "github.com/src-d/metadata-retrieval",
+		BinaryName:  "cmd",
```

Comment:
hm, why binary name is cmd?

Reply:
I've opened an [issue](https://github.com/src-d/metadata-retrieval/issues/85) for that

Reply:
hm, this might be a problem taking into account your [PR](https://github.com/src-d/metadata-retrieval/pull/90), we should wait for this fix before creating the regression branch.
don't you want to discuss this part with applications team and fix it yourself? this will reduce the waiting time for both of this and `regression.yml` PRs

Reply:
Yeap on it